sporkfish/searcher.py

```diff
@@ -411,12 +474,13 @@ def do_search_with_info(
         move = chess.Move.null()
 
         # Iterative deepening
+        # This conducts a fixed-depth search for depths ranging from 1 to max_depth.
```

Comment:
nice comment - suggest in the future, when we have wiki pages - we can just change the comments to point to corresponding wiki pages.